research-hub-web/src/app/components/home/browse/browse.component.ts

```diff
@@ -32,6 +33,45 @@ export class BrowseComponent implements OnInit, OnDestroy {
     this.mediaSub = this.media.media$.subscribe((change: MediaChange) => {
       this.updateCols(change.mqAlias);
     });
+
+    enum OptionType {
+      ResearchActivity = 1,
+      Category,
+      Menu
+    }
+
+    enum CategoryId {
+      All = 1,
+      Support,
+      Equipment,
+      Training,
+      Software,
+      Facilities,
+      Guide,
+      Person,
+      Policies,
+      Articles,
+      SubHubs
+    }
+
+    this.categoryOptions = [
```

Comment:
I reaaaaaaaaaaaallly hate all this 😡 
Not your fault though. I just don't like that we're hardcoding all this, I think in the future we should try and store this information in Contentful in some sorta metatype.